packages/fluentui/react-northstar/test/specs/commonTests/isConformant.tsx

```diff
@@ -408,8 +427,13 @@ export default function isConformant(
           [listenerName]: handlerSpy,
           [EVENT_TARGET_ATTRIBUTE]: true,
         };
-
-        const component = mount(<Component {...props} />);
+        const ComponentHandled = passesUnhandledPropsTo;
+        const component = passesUnhandledPropsTo
+          ? mount(<ComponentHandled {...props} />)
+          : mount(<Component {...props} />);
```

Comment:
This should be still:

```suggestion
        const component = mount(<Component {...props} />);
```

Otherwise it will execute tests on `Slider` instead of `FormSlider`

Reply:
If doesn't not do like that we get error because it will not find the handled events in the proptypes of `FormButton` form example and it will throw error. Other way to make it work is to set 

```
FormButton.propTypes = {
...,
onClick:...
}
```

And then pass it directly on the `slotProps` to control